arrow-libs/core/arrow-core/src/commonMain/kotlin/arrow/core/NonEmptyList.kt

```diff
@@ -179,7 +179,7 @@ public value class NonEmptyList<out A> @PublishedApi internal constructor(
     else -> head
   }
 
-  @Suppress("OVERRIDE_BY_INLINE")
+  @Suppress("OVERRIDE_BY_INLINE", "NOTHING_TO_INLINE")
```

Comment:
@serras, so is this is actually inlining?

I know Kotlin Std has `@InlineOnly`, and afaik that forces inlining and removes the method from the binary. At least that's my understanding. So I am wondering, if that exists and my understanding is correct does the compiler ignore this `inline` or does it still forces inline (but keep it in the binary?)?

Reply:
I'm not sure. The error I was getting was "performance of `inline` is not significant" (even though the error is called "nothing to inline"). I think we should keep those `inline`, it feels like each step is not very significant, but in a big code this all adds up.